lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -243,6 +279,45 @@ public void OnNext(IFailedEvaluator value)
         }
 
         /// <summary>
+        /// This is to remove failed context from _activeContexts
+        /// More details will be implemented when working on REEF-1251
+        /// </summary>
+        /// <param name="value"></param>
+        private void RemovedFailedContext(IFailedEvaluator value)
```

Comment:
Methods like this make me wonder whether we should introduce a `ContextManager` in this series of refactorings earlier rather than later. We can merge this, but as @dkm2110 pointed out, it will move around in the next PR again. It might be better to first port the current behavior to the new structure and then add behavior then to first add new behavior and then to restructure it.